utils/utils.go

```diff
@@ -3,15 +3,27 @@ package utils
 import (
 	"encoding/json"
 	"fmt"
+	"math/rand"
 	"strconv"
+	"strings"
+	"time"
 
 	oscgo "github.com/outscale/osc-sdk-go/v2"
 	"github.com/spf13/cast"
 
 	"github.com/aws/aws-sdk-go/aws"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
 )
 
 // PrintToJSON method helper to debug responses
+const (
+	ResourceNotFound string = "doesn't exist."
```

Comment:
Specify that this is a APi error message with for example ResourceNotFoundErrorMessage